nusight2/build_scripts/build.ts

```diff
@@ -15,6 +13,20 @@ const opts: ConfigOptions = {
   sourceMap: isContinuousIntegration ? false : 'source-map',
   rootDir: path.join(__dirname, '..'),
 }
+
 webpack([getClientConfig(opts), getServerConfig(opts)]).run((err, stats) => {
-  stats && process.stdout.write(stats.toString({ colors: true }) + '\n')
+  if (stats) {
+    process.stdout.write(stats.toString({ colors: true }) + '\n')
+    if (stats.hasErrors()) {
+      process.exitCode = 1
+    }
+  }
+  if (err) {
+    process.stderr.write(err.message)
+    if (err.stack) {
+      process.stderr.write(err.stack)
+    }
+    process.exitCode = 1
+  }
```

Comment:
Is idiomatic to handle `err` first thing in the callback, so I'd move this up, with an early return if there's an error, like the example shows here: https://webpack.js.org/api/node/#error-handling.